### PR TITLE
PMM-10677: Implement parsing value of metric to json

### PR DESCRIPTION
**Pull request by dliakhov**
PMM-10677: Implement parsing value of metric to json

Explanation how it should work you can find here https://confluence.percona.com/pages/viewpage.action?pageId=138486089. See there info about field `data_json`.

- [ ] Links to other linked pull requests (optional).

### CODE REVIEW

File: managed/services/telemetry/config.default.yml [hunk 1]
```diff
@@ -49,6 +49,20 @@ telemetry:
       - metric_name: "pmm_server_updates_disabled"
         column: "updates_disabled"
 
+  - id: PMMAgentRandomInfo
+    source: PMMDB_SELECT
+    query: agent_type, status from agents
+    summary: "PMMAgentRandomInfo"
+    json_format:
```
Comment (ritbl) [inline managed/services/telemetry/config.default.yml:56]: nit: it would be good to follow naming schema. e.g. `data_json`

Comment (dliakhov) [reply managed/services/telemetry/config.default.yml]: changed to data_json

File: managed/services/telemetry/config.default.yml [hunk 2]
```diff
@@ -49,6 +49,7 @@ telemetry:
       - metric_name: "pmm_server_updates_disabled"
         column: "updates_disabled"
 
+
```
Comment (ritbl) [inline managed/services/telemetry/config.default.yml:52]: nit: no need for additional space

Comment (dliakhov) [reply managed/services/telemetry/config.default.yml]: Removed this line

**Pull request closed without merging**